addons/mail/models/mail_activity.py

```diff
@@ -76,6 +76,7 @@ def _default_activity_type_for_model(self, model):
     activity_category = fields.Selection(related='activity_type_id.category', readonly=True)
     activity_decoration = fields.Selection(related='activity_type_id.decoration_type', readonly=True)
     icon = fields.Char('Icon', related='activity_type_id.icon', readonly=True)
+    plan_id = fields.Many2one('mail.activity.plan', string='Plan')
```

Comment:
ondelete set null probably, as we don't want to loose activities nor prevent plan removal just for that (it is mainly logging purpose)

Reply:
And copy=False also